swarm/src/behaviour.rs

```diff
@@ -191,6 +192,35 @@ pub trait NetworkBehaviour: 'static {
         vec![]
     }
 
+    // TODO: Make sure this and all the methods below are really implemented across all behaviours.
+    fn review_pending_connection(
```

Comment:
I am not fully sold on the "review" terminology but I also can't think of something better off the top of my head.

Reply:
Not a fan of "review" either. Grateful for naming suggestions!